### DISCUSSION TITLE
Feature request: CRUD for Prompt API

### DISCUSSION THREAD

**Discussion by barseghyanartur** [langfuse/langfuse#4430] [Ideas]
### Describe the feature or potential improvement

At the moment, it's not possible to perform a clean-up of old prompts programmatically.

It would be beneficial to implement full CRUD functionality for prompts, as it would address the shortcomings of LangFuse in more complex workflows that deviate from standard LangFuse practices.

Finally, in addition to the standard `create`, `read`, `update`, and `delete` operations, a couple of specialized endpoints would be useful too:

- Remove all prompts by `name` except the latest production version.
- Remove all prompts by `name` and `tag`.

Naturally, reflecting the changes in the SDK is desirable too.

**Comment by albcunha** [comment]
I agree this would be a great improvement. I was surprise to see that the api for this is only CR.
